cg/meta/workflow/prepare_fastq.py

```diff
@@ -35,14 +35,11 @@ def get_compression_objects(self, case_id: str) -> List[CompressionData]:
     def is_spring_decompression_needed(self, case_id: str) -> bool:
         """Check if spring decompression needs to be started"""
         compression_objects = self.get_compression_objects(case_id=case_id)
-        for compression_object in compression_objects:
-            if (
-                self.crunchy_api.is_compression_pending(compression_object)
-                or compression_object.pair_exists()
-            ):
-                continue
-            return True
-        return False
+        return any(
```

Comment:
Nice! Double check that the logic is covered by a good unit test so we do not accidentally start doing a lot of compressions

Reply:
I think this funciton is already tested in [PrepareFastqAPI](https://github.com/Clinical-Genomics/cg/blob/master/tests/meta/workflow/test_prepare_fastq_api.py), all I did was to make the function more "pythonic" or something 💯